stdlib/src/builtin/hash.mojo

```diff
@@ -63,14 +63,16 @@ fn _HASH_SECRET() -> Int:
     return ptr.bitcast[Int]()[0]
 
 
-fn _initialize_hash_secret(payload: Pointer[NoneType]) -> Pointer[NoneType]:
+fn _initialize_hash_secret(
+    payload: UnsafePointer[NoneType],
+) -> UnsafePointer[NoneType]:
     var secret = random.random_ui64(0, UInt64.MAX)
-    var data = Pointer[Int].alloc(1)
-    data.store(int(secret))
+    var data = UnsafePointer[Int].alloc(1)
+    initialize_pointee_move(data, int(secret))
```

Comment:
I think we don't need this for reg types.

Reply:
What code change would you recommend?

Reply:
Done!